Sources/ProjectDescription/Dependency.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Dependency.swift
+//
+//
+//  Created by Facundo Menzella on 27/09/2020.
+//
+
+import Foundation
+
+public struct Dependency: Codable, Equatable {
```

Comment:
Since this model is part of `ProjectDescription`, which means it's user-facing, documentation here is very important so that users can get it while they are editing their manifests on Xcode. I'd documment the class, the properties, and the constructor.